tools/LoadPackage.groovy

```diff
@@ -96,7 +96,7 @@ class LoadPackage {
 
     def run(File package_dir) {
         def target_dir_path = ""
-        def bucket = ""
+        def package_buckets_list
```

Comment:
a little bit confusing.. why not buckets_list ?

Reply:
yes

Reply:
To improve readability.. it could be better to split the run function into specialized smallest ones.
Code sections could being boxed 

from l101 to l114  ( unzipPAckage(xx) )
from l124 to l140 ( populateDataspace(xxx))
from l153 to l287 ( populateBucket(xxxx) )
from l191 to l286 (pushObject(xxx) )
from l231 to l248 (populateTemplateDir(xxx))